pkg/expapi/validation/validation.go

```diff
@@ -65,3 +65,11 @@ func ValidateHorizontalPodAutoscalerUpdate(newAutoscler, oldAutoscaler *expapi.H
 	allErrs = append(allErrs, validateHorizontalPodAutoscalerSpec(newAutoscler.Spec)...)
 	return allErrs
 }
+
+func ValidateDerivedNodeMetrics(metrics *expapi.DerivedNodeMetrics) errs.ValidationErrorList {
+	return nil
```

Comment:
Fill these out with at least the minimal metadata validation for both regular and updates.